Nodejs/NodejsTools.sln

```diff
@@ -91,6 +94,7 @@ Global
 		{240A6D2B-1982-4BC5-B1EA-C6CB0492329D}.Release|x86.Build.0 = Release|x86
 		{D092D54E-FF29-4D32-9AEE-4EF704C92F67}.Debug|x86.ActiveCfg = Debug|x86
```

Comment:
Please revert the changes in this file. I don't think any updates are needed based on the rest of the changes.